lib/wickedbase.pm

```diff
@@ -737,6 +738,51 @@ sub record_console_test_result {
     $self->write_resultfile($filename, $content);
 }
 
+sub check_logs {
+    my $self = shift;
+    my @units = qw(wickedd-nanny wickedd-dhcp4 wickedd-dhcp6 wicked wickedd);
+    my $default_exclude = 'wickedd=process \d+ has not exited yet; now doing a blocking waitpid';
+    $default_exclude .= ',wickedd-dhcp6=Link-local IPv6 address is marked duplicate:';
+    $default_exclude .= ',wickedd-nanny=: device has been deleted';
+    $default_exclude .= ',wickedd-dhcp4=unable to confirm lease';
+    $default_exclude .= ',wickedd-nanny=: call to org.opensuse.Network.Interface.waitLinkUp\(\) failed: General failure';
+    $default_exclude .= ',wickedd-nanny=: call to org.opensuse.Network.Interface.waitLinkUp\(\) failed: Object does not support requested method';
+    $default_exclude .= ',wickedd-nanny=: failed to bring up device, still continuing';
+    $default_exclude .= ',wickedd=error retrieving tap attribute from sysfs';
+
+    my $exclude_var = get_var(WICKED_CHECK_LOG_EXCLUDE => $default_exclude);
```

Comment:
we need to add this variables to `variables.md`

Reply:
Agree but within this PR? I mean there is none of the `WICKED_*` variables in there...

Reply:
ok